examples/mobile/app/build.gradle

```diff
@@ -0,0 +1,81 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    namespace 'org.bevyengine.example'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "org.bevyengine.example"
+        minSdk 21
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            minifyEnabled false
+            //packagingOptions {
+            //    doNotStrip '**/*.so'
+            //}
+            //debuggable true
+        }
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    sourceSets {
+        main {
+            res {
+                srcDir(layout.projectDirectory.dir("../../../assets/android-res/"))
+            }
+            // assets are copied to the normal assets directory by the copyAssets task during assembly stage
+        }
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.core:core:1.9.0'
+    implementation 'androidx.appcompat:appcompat:1.6.1'
+    implementation 'com.google.android.material:material:1.7.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+
+    // To use the Android Frame Pacing library
+    //implementation "androidx.games:games-frame-pacing:1.9.1"
+
+    // To use the Android Performance Tuner
+    //implementation "androidx.games:games-performance-tuner:1.5.0"
+
+    // To use the Games Activity library
+    //implementation("androidx.games:games-activity:3.0.2")		// 3.0.2 results in an UnsatisfiedLinkError due to the change in onTouchEventNative method signature
+    implementation("androidx.games:games-activity:2.0.2")
+
+    // To use the Games Controller Library
+    //implementation "androidx.games:games-controller:1.1.0"
+
+    // To use the Games Text Input Library
+    //implementation "androidx.games:games-text-input:1.1.0"
```

Comment:
SOme of these are outdated. Now, it doesn't make much sense to keep chasing the latest versions for example code, but I would recommend adding a comment that points this out.